src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/netcore/NetCoreLegacyCachePersistance.cs

```diff
@@ -1,46 +0,0 @@
-﻿//----------------------------------------------------------------------
```

Comment:
All these netcore files were leftover from the ADAL+MSAL unified repo. They are not in use.